result

```diff
@@ -0,0 +1 @@
+/nix/store/ri02df163pk8qxaqdj9f620zfg0hm72k-fedimintd-0.0.1
```

Comment:
I think that slipped in just delete or ignore and do `commit --amend`

Reply:
good catch!

Reply:
We `.gitignore /result` folder, so it's really sus that this made it through even through rebase!